pulsar-broker/src/main/java/org/apache/pulsar/PulsarBrokerStarter.java

```diff
@@ -139,6 +140,15 @@ private static boolean argsContains(String[] args, String arg) {
                 brokerConfig = loadConfig(starterArguments.brokerConfigFile);
             }
 
+            int maxFrameSize = brokerConfig.getMaxMessageSize() + (10 * 1024);
+            if (maxFrameSize < 0) {
+                throw new IllegalArgumentException("Max message size need smaller than 5233640 bytes");
+            }
+            if (maxFrameSize > DirectMemoryUtils.maxDirectMemory()) {
```

Comment:
This check is not going to be very useful. Even if we have 1 single message of a size comparable to maxDirectMem, we'd be already in trouble

Reply:
Ok, will fix it as `>=`